public/index.php

```diff
@@ -231,6 +231,13 @@ function displayUrl($url, $main) {
         <p>Will result in the following cookie: <samp /></p>
         <input type="submit" />
       </form>
+      <p>Credentials:
+      <label><input type="radio" name="credentials" value="same-origin" checked />same-origin</label> 
+      <label><input type="radio" name="credentials" value="include" />include</label>
+      <label><input type="radio" name="credentials" value="omit" />omit</label>
+      <small><a href="https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#including_credentials">See MDN docs for fetch()#Include credentials</a> </small>
+      </p>
+      <button onClick="callFetch()">Call fetch()</button>
```

Comment:
When I try every time I click on the button page is refreshed.
I guess it because of the missing `type` attribute: `type="button"`

Reply:
I see now. We can say this is intentional. it comes from "clicking" the reload button after server response 
```js
      document.querySelector('a.reload').click()
```

We do want to show newly set cookies after server response. How can we avoid a full page reload here?